### ISSUE TITLE
Reducing noise in evaluate by averaging

### ISSUE THREAD

**Issue by baggepinnen** [alan-turing-institute/MLJ.jl#406]
To reduce the noise when evaluating a model on a small dataset I frequently implement the following pattern
```julia
mean(1:10) do i
    e = MLJ.evaluate(model, Xt, Ylabel, resampling=CV(nfolds=3, shuffle=true), measure=MLJ.auc, verbosity=1, check_measure=true)
    e.measurement[]
end
```
Adjusting `nfolds` is not really a solution, especially when the dataset is small and unbalanced. 
An additional keyword arg `average=10` or `repeat=10` or something like that would be a nice feature to have.

**Comment by ablaom** [comment]
Thanks for that input!

I understand this is certainly a common pattern.  I'm not sure I understand precisely your suggestion for implementation. If, for example, `nfolds=2` and  `repeat=3` then, what exactly should the values of `e.per_fold` be? For simplicity, if there is only a single measure specified, so that `e.per_fold` is a vector `[v,]`, then does `v` have 6 elements or still just 2 (one for each fold)? Or something else?

**Comment by baggepinnen** [comment]
My thoughts were that there would be 6 folds in total since if `shuffle=true` the folds will be different each time. Every fold would still consist of half of the data though. `v` would thus have 6 elements.

I am usually not very interested in the folds themselves, but rather the statistics of the measure over many folds.

**Comment by ablaom** [comment]
Agreed, this makes sense. Will implement.

**Comment by ablaom** [comment]
Added to MLJBase. Will be available in MLJ 0.7.0.
